### DISCUSSION TITLE
你好，怎样将上传的视频变成行内元素？

### DISCUSSION THREAD

**Discussion by xiewanqin** [wangeditor-next/wangEditor-next#386] [General]
## 问题描述

*定义了一个插件，代码如下*
```
function withVideo<T extends IDomEditor>(editor: T) {
  const { isInline, isVoid } = editor
  const newEditor = editor

  newEditor.isInline = (elem) => {
    const type = DomEditor.getNodeType(elem)

    if (type === 'video') return true // 针对 type: video ，设置为 inline
    return isInline(elem)
  }

  return newEditor // 返回 newEditor ，重要！！！
}
Boot.registerPlugin(withVideo)
```
请问还需要做什么，目前我这样写插入视频不能显示了

## wangEditor 版本

*5.6.12*

## 是否查阅了文档 ？

（文档链接 [https://cycleccc.github.io/docs/](https://cycleccc.github.io/docs/) ）

*是*

**Comment by cycleccc** [comment]
你需要自定义个一个具有浮动属性的 video。
以下插件是有浮动属性的图片，仿照这改一个 video 的试试。
https://github.com/cycleccc/wangEditor-next/tree/master/packages/plugin-float-image

**Comment by cycleccc** [comment]
行内或浮动视频需通过自定义 video 元素扩展实现，可参考 float-image 插件。本讨论现关闭。
